src/ImageSharp.Web/Caching/PhysicalFileSystemCache.cs

```diff
@@ -62,15 +62,20 @@ public class PhysicalFileSystemCache : IImageCache
         /// <summary>
         /// Initializes a new instance of the <see cref="PhysicalFileSystemCache"/> class.
         /// </summary>
-        /// <param name="environment">The hosting environment the application is running in</param>
-        /// <param name="bufferManager">An <see cref="IBufferManager"/> instance used to allocate arrays transporting encoded image data</param>
-        /// <param name="options">The middleware configuration options</param>
-        public PhysicalFileSystemCache(IHostingEnvironment environment, IBufferManager bufferManager, IOptions<ImageSharpMiddlewareOptions> options)
+        /// <param name="environment">The hosting environment the application is running in.</param>
+        /// <param name="bufferManager">An <see cref="MemoryAllocator"/> instance used to allocate arrays transporting encoded image data.</param>
+        /// <param name="options">The middleware configuration options.</param>
+        public PhysicalFileSystemCache(IHostingEnvironment environment, MemoryAllocator bufferManager, IOptions<ImageSharpMiddlewareOptions> options)
```

Comment:
@JimBobSquarePants I forgot to rename the parameter `bufferManager` on a plenty of places.

Reply:
Now worries, I'll fix that.